treeherder/etl/job_loader.py

```diff
@@ -300,4 +300,4 @@ def _get_validated_jobs_by_project(self, jobs_list):
         return validated_jobs
 
     def _to_timestamp(self, datestr):
-        return time.mktime(parser.parse(datestr).timetuple())
+        return calendar.timegm(parser.parse(datestr).utctimetuple())
```

Comment:
I know, I was salivating over that function of python 3.3...  :)  Wishing we could move to Python 3 at some point before long.  I actually ran into that and thought, "oh, that's perfect!" then I noticed the version of the docs I was looking at...  ;)